SparkPost/2web2ui
# FE-1366 - Alert is displayed only once

FE-1366 - Alert is displayed only once

### What Changed

- Alert on verification of domain through token will display the success/error alert only once.

### How To Test

Steps to reproduce the bug:

- Verify a domain by email.

- Click on the link in the email notification.

- After being redirected to the app, and if the domain is successfully verified, navigate away from the Domains list page and back

- ~Subsequent visits to the list page display the same Snackbar~ subsequent visits should display snackbar only once.

### To Do
